### ISSUE TITLE
弱弱的问一个问题，怎么取出一个tensor的值，类似于tensor.data的操作？

### ISSUE THREAD

**Issue by Chenfei-Kang** [PaddlePaddle/Paddle#33002]
为使您的问题得到快速解决，在建立Issues前，请您先通过如下方式搜索是否有相似问题:【搜索issue关键字】【使用labels筛选】【官方文档】

如果您没有查询到相似问题，为快速解决您的提问，建立issue时请提供如下细节信息：
- 标题：简洁、精准概括您的问题
- 版本、环境信息：
    1）PaddlePaddle版本：请提供您的PaddlePaddle版本号，例如1.1或CommitID
    2）CPU/GPU：如果您使用GPU训练，请提供GPU驱动版本、CUDA和cuDNN版本号
    3）系统环境：请您描述系统类型、版本，例如Mac OS 10.14
    4）Python版本号
    5）显存信息
 注：您可以通过执行[summary_env.py](https://github.com/PaddlePaddle/Paddle/blob/develop/tools/summary_env.py)获取以上信息。
- 复现信息：如为报错，请给出复现环境、复现步骤
- 问题描述：请详细描述您的问题，同步贴出报错信息、日志/代码关键片段

Thank you for contributing to PaddlePaddle.
Before submitting the issue, you could search issue in the github in case that there was a similar issue submitted or resolved before.
If there is no solution,please provide us with the following details :
**System information**
-PaddlePaddle version （eg.1.1）or CommitID
-CPU: including CPUMKL/OpenBlas/MKLDNN version
-GPU: including CUDA/cuDNN version
-OS Platform and Distribution(eg.Mac OS 10.14)
-Python version 
Note: You can get most of the information by running [summary_env.py](https://github.com/PaddlePaddle/Paddle/blob/develop/tools/summary_env.py). 
**To Reproduce**
Steps to reproduce the behavior
**Describe your current behavior**
**Code to reproduce the issue**
**Other info / logs**

**Comment by wanghuancoder** [comment]
Tensor的使用基本和numpy一样，你可以试试类似于numpy的方法，比如：tensor[i,j]。
同时，Tensor可以转成numpy，tensor.numpy()。

**Comment by Chenfei-Kang** [comment]
> Tensor的使用基本和numpy一样，你可以试试类似于numpy的方法，比如：tensor[i,j]。
> 同时，Tensor可以转成numpy，tensor.numpy()。

感谢

**Comment by hua-zi** [comment]
> Tensor的使用基本和numpy一样，你可以试试类似于numpy的方法，比如：tensor[i,j]。 同时，Tensor可以转成numpy，tensor.numpy()。

请问可以使用 `tensor.detach()` 来复现 torch的 `tensor.data` 吗
